helix-term/src/ui/statusline.rs

```diff
@@ -406,7 +407,7 @@ where
     write(context, format!(" {} ", file_type), None);
 }
 
-fn render_file_name<F>(context: &mut RenderContext, write: F)
+fn render_file_path<F>(context: &mut RenderContext, write: F)
```

Comment:
I liked the `FileBaseName` name from the discussion in #5279. The rename here would be a breaking change that would affect any custom statusline.left configs people have

Reply:
Added another commit to address this.